libs/SalesforceSDK/src/com/salesforce/androidsdk/app/SalesforceSDKManager.kt

```diff
@@ -234,6 +232,16 @@ open class SalesforceSDKManager protected constructor(
      */
     var loginViewModelFactory = LoginViewModel.Factory
 
+    /**
+     * Asynchronously retrieves the app config for the specified login host.  If not set the values
+     * found in the BootConfig file will be used for all servers.
+     */
+    var appConfigForLoginHost: suspend (server: String) -> OAuthConfig = {
```

Comment:
On iOS, if the block is not set OR returns nil, then the default config is being used.
See comment [here](https://github.com/forcedotcom/SalesforceMobileSDK-iOS/blob/01a3ac3ffbc0c1e144cbd370e750fa4e7d715e84/libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Common/SalesforceSDKManager.h#L326) and code [here](https://github.com/forcedotcom/SalesforceMobileSDK-iOS/blob/01a3ac3ffbc0c1e144cbd370e750fa4e7d715e84/libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Common/SalesforceSDKManager.m#L1006)